vault addCollateral proposal updates

0xpatrickdev opened the pull request:
### Reproducable Bundles:

I am working off of this:  https://github.com/Agoric/agoric-sdk/compare/master...0xpatrickdev:agoric-sdk:pc/statom-vault-proposal
 -  node v16.18.1,  go1.20.2 darwin/arm64,  [agoric-sdk#9d4eaad](https://github.com/Agoric/agoric-sdk/commit/9d4eaadaa3caa031a0410f747ff06a4855ece40a)


## What was added

- removed `economicCommitteeCreatorFacet ` and `contractGovenor` from oracle permit: https://github.com/Agoric/agoric-sdk/commit/b414246ceb9770f3a4c1a840ca1d19c1c8ed32f4  (see #5)

- updated README instructions to suggest single proposal  (see #6)

- use `STATOM1` in proposal for now (see #1, #10) 

- update oracle operator addresses (decentral-devnet-config + pc address) (see #3)

- docs for oracle flow, and REPL validation


## What is missing

- I attempted to tackle #7, `restoreRef` for `priceAggregator` and  `scaledPriceAuthority`, but couldn't quite figure this out. I am not sure yet what needs to be changed so we have access to `restoreRef` in `add-collateral-core.js`.


## Testing Status

I included `ibc/toystatom` in all the places that have `ibc/toyatom` in cosmic-swingset/Makefile, and this enables me to access test statom1 in the repl and other wallets.

~~I am (seemingly) able to submit oracles prices using the agops cli, but am getting a `maxDebtFor called before a collateral quote was available` error when opening an stATOM vault. This error occurs when `storedCollateralQuote` is undefined, so it seems I may be missing a step.~~ Oracle prices must be sent from 2-3 addresses/operators, before the price registers. Once I included this in the flow, things are working as expected 🎉


<img width="957" alt="image" src="https://github.com/0xpatrickdev/agoric-vault-collateral-proposal/assets/11021913/47ee80eb-a259-48d8-ad68-ba0b1afc7e51">
<img width="747" alt="image" src="https://github.com/0xpatrickdev/agoric-vault-collateral-proposal/assets/11021913/2ff86bf0-9dda-4f23-86eb-bd5c28962f7f">

0xpatrickdev committed:
feat: updates bundles with latest build

- use STATOM1 instead STATOM for testing
- limit powers in oracle proposal (economicCommitteeCreatorFacet, contractGovernor) see #5
- use testnet oracle address (+ my own) see #3
- bundles: use node v16.18.1, go 1.20.2, agoric-sdk master#9d4eaad see #9

0xpatrickdev committed:
docs: submit multiple scripts in one proposal

0xpatrickdev committed:
docs: update readme

dckc commented:
This is good progress. Several of the fixes look good.

If they were in their own PR(s), I would approve them, but I'm not sure what to do with this whole PR.

dckc reviewed (approved):
ok. I get it now. this addresses the issues in "what was added"

0xpatrickdev committed:
feat(docs): oracle and wallet provision steps

- guidance on how to provision + fund smart wallet on local
- explain how oracle prices must be sent from 2-3 addreses, depending the chain configuration
